src/main/java/nl/tudelft/dnainator/ui/drawables/ClusterDrawable.java

```diff
@@ -0,0 +1,41 @@
+package nl.tudelft.dnainator.ui.drawables;
+
+import java.util.List;
+
+import javafx.scene.Group;
+import javafx.scene.paint.Color;
+import javafx.scene.shape.Circle;
+import javafx.scene.text.Text;
+import nl.tudelft.dnainator.core.SequenceNode;
+
+/**
+ * The {@link ClusterDrawable} class represents the mid level object in the viewable model.
+ */
+public class ClusterDrawable extends Group {
+	private static final int CLUSTER_SIZE = 3;
+	private List<SequenceNode> clustered;
+	private Circle c;
```

Comment:
variable names should represent what they are/represent. Consider renaming this to `cluster` or something similar